src/main/kotlin/com/hibob/feedback/dao/FeedbackDao.kt

```diff
@@ -61,4 +61,14 @@ class FeedbackDao(private val sql: DSLContext) {
             .fetchOne(feedbackMapper)
             ?: throw BadRequestException("Feedback not found")
     }
+
+    fun getStatus(feedbackId: UUID, activeUser: ActiveUser): Status {
+        return sql.select(feedbackTables.status)
+            .from(feedbackTables)
+            .where(feedbackTables.feedbackId.eq(feedbackId))
+            .and(feedbackTables.companyId.eq(activeUser.companyId))
+            .and(feedbackTables.employeeId.eq(activeUser.employeeId))
```

Comment:
the only person who can look at the status of a request is the person who created it?

Reply:
Status Checking API: Provide an API endpoint for employees to check the status of their submitted feedback (if not anonymous).

the spec sounds like that

Reply:
I thought that this API is for employee only and the Filter is for the HR/ admin